backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -222,13 +223,16 @@ def _generate_privilege_record(
         jurisdiction_postal_abbreviation: str,
         license_expiration_date: date,
         compact_transaction_id: str,
+        original_issuance_date: date | None = None,
     ):
+        today = datetime.now(tz=UTC).date()
         privilege_object = {
             'providerId': provider_id,
             'compact': compact_name,
             'jurisdiction': jurisdiction_postal_abbreviation.lower(),
             'status': 'active',
-            'dateOfIssuance': datetime.now(tz=UTC).date(),
+            'dateOfIssuance': original_issuance_date if original_issuance_date else today,
```

Comment:
Accessibility: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

<img width="577" alt="image" src="https://github.com/user-attachments/assets/6cfe0a84-837d-4bd4-989c-1ee4a5834f6d">

@jsandoval81